pnpm-workspace.yaml

```diff
@@ -14,6 +14,7 @@ catalog:
   rimraf: "^5.0.10"
   typescript: "^5.7.2"
   undici: "^5.28.5"
+  unenv: "2.0.0-rc.8"
```

Comment:
we should not do that before unenv is stable (2.0)

Reply:
The thing here is that we would need to test both Vite + Wrangler when updating unenv and I'm afraid it might be too much constraint while things are still moving (fast).

Once unenv@2.0 lands, the changes should have a smaller extent and moving to a catalog at that point might make more sense. We can also try and opt-out if needed.